README.md

```diff
@@ -1 +1,156 @@
-# opencraft
+OpenCraft
+=========
+
+Install
+-------
+
+Instructions based on Ubuntu 14.04.
+
+Install the system package dependencies & virtualenv:
+
+```
+$ sudo apt-get install `cat debian_packages.lst`
+$ pip3 install --user virtualenv && pip3 install --user virtualenvwrapper
+```
+
+Ensure you load virtualenv with Python 3 in `~/.bashrc`:
+
+```
+export PATH="$PATH:$HOME/.local/bin" VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3
+source $HOME/.local/bin/virtualenvwrapper.sh
+```
+
+Then reload `~/.bashrc`, create the virtual env and install the Python requirements:
+
+```
+$ . ~/.bashrc
+$ mkvirtualenv -p /usr/bin/python3 opencraft
+$ pip install -r requirements.txt
+```
+
+
+Configure
+---------
+
+Create an `.env` file at the root of the repository or set environment variables, customizing the
+settings from `opencraft/settings.py` which are loaded via `env()`.
+
+
+Run
+---
+
+To run the development server:
+
+```
+$ make rundev
+```
+
+Then go to:
+
+* User interface: [http://localhost:5000/](http://localhost:2000/)
+* API: [http://localhost:5000/api/](http://localhost:2000/api/)
+* Admin: [http://localhost:5000/admin/](http://localhost:2000/admin/)
```

Comment:
Good catch - fixed. I have also made the ports/hosts configurable, as you suggested.